onnxruntime/core/providers/partitioning_utils.h

```diff
@@ -18,27 +18,73 @@ class Node;
 
 namespace utils {
 
+/**
+Called to check whether a node is supported.
+
+@param node The node to check.
+
+@return Whether the node is supported.
+*/
+using IsNodeSupportedFn = std::function<bool(const Node& node)>;
```

Comment:
what does all the `Fn` in the name stand for?

Reply:
function